Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -38,14 +38,18 @@ class Digitizer
   int mEventID = 0;
   int mSrcID = 0;
 
-  float mWion; //  Ionization potential
+  float mWion;   //  Ionization potential
+  bool mSDigits; // true :convert signals to summable digits, false by default
```

Comment:
I don't see a setter for this variable.

Reply:
That variable is defined in the constructor, but I can add the setter and do it that way too.